database/src/main/runs/get_flow_additional_data.sql

```diff
@@ -0,0 +1,85 @@
+/*
+ * Copyright 2021 ABSA Group Limited
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *     http://www.apache.org/licenses/LICENSE-2.0 *
```

Comment:
```suggestion
 *     http://www.apache.org/licenses/LICENSE-2.0 
 *
```

Reply:
Removed.

Reply:
https://www.apache.org/licenses/LICENSE-2.0#apply

To be perfectly precise. There should be a new line between the link as outlined in the `apply` section of the license

@AB006J9 applies to your PR as well.

Reply:
I see. New line added.